.gitignore

```diff
@@ -1,7 +1,6 @@
 pkg/
 Gemfile.lock
 Gemfile.local
-vendor/
```

Comment:
@KosherSalt I think we have a different ticket to evaluate that. I agree that it is something we need to fix, but it probably won't be part of this current PR.